badges/casual_instructors_2024.json

```diff
@@ -0,0 +1,381 @@
+{
+    "title": "Casual Instructors - 2024",
+    "subtitle": "Semester 3 and 5",
```

Comment:
What should be the correct semster range ?

Reply:
Semester 2, 4 and 6